mlir/lib/Dialect/Vector/Transforms/VectorEmulateNarrowType.cpp

```diff
@@ -102,6 +129,36 @@ static FailureOr<Operation *> getCompressedMaskOp(OpBuilder &rewriter,
   return newMask;
 }
 
+static std::optional<int64_t>
+getIntraVectorOffset(ConversionPatternRewriter &rewriter, Location loc,
+                     const memref::LinearizedMemRefInfo linearizedInfo) {
+  auto foldedIntraVectorOffset = getValueOrCreateConstantIndexOp(
+      rewriter, loc, linearizedInfo.intraVectorOffset);
+  return getConstantIntValue(foldedIntraVectorOffset);
+}
```

Comment:
I think we don't need the method. You can just replace all the usages with `getConstantIntValue(linearizedInfo.intraVectorOffset)`.

Reply:
done.